### PR TITLE
Add `paymentTerms` field when creating a cost center.

### PR DESCRIPTION
**Pull request by rayssamfreitas**
#### What problem is this solving?

Adjusts to send the `paymentTerms` field when creating an organization.

#### How to test it?

You can test by creating an organization and also passing a `paymentTerms`.

[Workspace](https://jwp14--jwpepperdev.myvtex.com/admin/graphql-ide)

#### Screenshots or example usage:
Captura de tela 2024-07-04 174226

### CODE REVIEW

**Comment by Rudge** [comment]
> Fails
> 🚫 	
> 
> We follow [keepachangelog](https://keepachangelog.com/) standards.
> 
> Please, change the file `CHANGELOG.md` adding a small summary of your changes under the `[Unreleased]` section. Example:
> 
> ```diff
>  ## [Unreleased]
> 
> + ### Fixed
> + - Issue with image sizing in the SKU Selector.
> ```
> 
> Possible types of changes:
> 
>     * `Added` for new features.
> 
>     * `Changed` for changes in existing functionality.
> 
>     * `Deprecated` for soon-to-be removed features.
> 
>     * `Removed` for now removed features.
> 
>     * `Fixed` for any bug fixes.
> 
>     * `Security` in case of vulnerabilities.
> 
> 
> 	Warnings
> ⚠️ 	
> 
> `package.json` was changed, but not `yarn.lock` (vice-versa) [:bulb: are you sure this is right?]
> 	Messages
> 📖 	❤️ Thanks!
> 📖 	
> 
> 🎉 PR additions = `5`, PR deletions = `2`
> 
> Generated by 🚫 [dangerJS](https://danger.systems/js) against [28d5ad6](https://github.com/vtex-apps/b2b-organizations-graphql/commit/28d5ad6a78c252face8bb6a4d9f88cd03443e90b)

@rayssamfreitas Could you update the changelog, please?

**Comment by rayssamfreitas** [comment]
@Rudge , updated!

File: node/resolvers/Mutations/CostCenters.ts [hunk 1]
```diff
@@ -70,6 +71,7 @@ const CostCenters = {
         phoneNumber,
         sellers,
         stateRegistration,
+        paymentTerms
```
Comment (enzomerca) [inline node/resolvers/Mutations/CostCenters.ts:74]: Our lint is pointing an error on this line: `Error:   74:21  error    Insert ',' prettier/prettier`
Could you fix it before we merge, please?
```suggestion
        paymentTerms,
```

**Review by enzomerca — DISMISSED** [review]
~~LGTM. We are going to merge/deploy it soon.~~

**Comment by enzomerca** [comment]
Upon further review, we believe this PR would cause the incorrect usage of a feature.

There is a feature that allows the Buyer Org Admin to enable/disable payment terms for a given cost center through the "My Account" section on the storefront. The cost center payment terms field, that can be updated through the updateCostCenter, should be used exclusively for that.
The misuse of this feature could cause unexpected behaviour when adding new payment terms to an existing buyer org through the Admin UI.

Could you please explain your use case so we can evaluate if this change makes sense or if there is an alternative way to achieve your needs?

**Comment by enzomerca** [comment]
Discussed offline, but it actually makes sense to add this parameter to the `createCostCenter` APIs.
The PR description/changelog info was saying `organization` but this change does not affect the `createOrganization` API.
I updated the PR description and changelog and I'm going to merge/deploy this change.

**Review by enzomerca — APPROVED** [review]

**Review by enzomerca — APPROVED** [review]

**Pull request merged**